examples/Billing/AddBillingSetup.cs

```diff
@@ -0,0 +1,205 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V3.Errors;
```

Comment:
Please make sure to change for [v4](https://ads-developers.googleblog.com/2020/07/announcing-v4-of-google-ads-api-beta.html) and wait for the client library to be released before merging this PR.

Reply:
We will merge this PR now, then pull it into v4 release by updating the version. I'm working with Andrew.

Reply:
SGTM